### PR TITLE
Describe benchmarks, minor tweaks

### PR DESCRIPTION
**Pull request by slightknack**

### CODE REVIEW

**Review by ShawSumma — APPROVED** [review]
Look fine! thank you!

File: README.md [hunk 1]
```diff
@@ -106,20 +112,43 @@ The best way to get familliar with MiniVM and its opcodes is to read through the
...
 
-### Recursive Fibonacci: Functions and Math
-> ![Fibonacci Runtime](fib.png)
+All benchmarks were run in hyperfine on a `2020 MacBook Air M1` with `8GB RAM` running `Big Sur 11.2.3`. The implementations we benchmarks are idiomatic and consistent between target benchmark languages. All benchmarks may be found in the [Paka repository](https://github.com/FastVM/paka/tree/main/bench) if you'd like to run them on your machine.
 
 ### Binary Trees: Allocations and GC
-> ![Binary Trees Graph](tree.png)
+![Binary Trees Graph](tree.png)
+
+As you can see, MiniVM (no JIT) beats luajit with the JIT on in this benchmark. MiniVM has a custom-built allocator and GC, which beats out luajit's slower modified version of `malloc`. MiniVM also is a hair faster that C for tree sizes above 13 (C is compiled ahead-of time, using `clang` with the `-Ofast` flag for best performance). For tree sizes less that 13, beats Node JS due to having a faster startup time. Overall, MiniVM's performance is about on par with JIT'd and compiled languages on this benchmark.
```
Comment (lqd) [inline README.md:125]: Here

```suggestion
As you can see, MiniVM (no JIT) beats luajit with the JIT on in this benchmark. MiniVM has a custom-built allocator and GC, which beats out luajit's slower modified version of `malloc`. MiniVM also is a hair faster than C for tree sizes above 13 (C is compiled ahead-of time, using `clang` with the `-Ofast` flag for best performance). For tree sizes less than 13, beats Node JS due to having a faster startup time. Overall, MiniVM's performance is about on par with JIT'd and compiled languages on this benchmark.
```

File: README.md [hunk 2]
```diff
@@ -106,20 +112,43 @@ The best way to get familliar with MiniVM and its opcodes is to read through the
...
+
+We compile this Paka straight to minivm bytecode, there isn't any subsequent optimization of the bytecode by hand.
 
 ## Roadmap
-- write assembler for minivm bytecode
-- reducde dependancies to libc functions
-- add types
-- improve performane
+- Write assembler for minivm bytecode.
+- Reducde dependencies to `libc` functions.
```
Comment (lqd) [inline README.md:150]: And here 
```suggestion
- Reduce dependencies to `libc` functions.
```

File: README.md [hunk 3]
```diff
@@ -106,20 +112,43 @@ The best way to get familliar with MiniVM and its opcodes is to read through the
...
 
-The most common way to get code running on minivm is to use [Paka](https://github.com/fastvm/paka). For those looking to try paka and minivm online use [XorI](https://fastvm.github.io/xori).
+The most common way to get code running on MiniVM is to use [Paka](https://github.com/fastvm/paka). For those looking to try paka and minivm online use [XorI](https://fastvm.github.io/xori).
 
 ## Benchmarks
+Taking benchmarks is hard. Benchmarks are fraught with peril and don't always tell the full story: if you want to know how your application will perform in a language, no benchmarks will be a substitute for that. We tried to be fair, methodical, and thorough in our benchmarking; despite this, remember to take these results with a grain of salt.
 
-### Recursive Fibonacci: Functions and Math
-> ![Fibonacci Runtime](fib.png)
+All benchmarks were run in hyperfine on a `2020 MacBook Air M1` with `8GB RAM` running `Big Sur 11.2.3`. The implementations we benchmarks are idiomatic and consistent between target benchmark languages. All benchmarks may be found in the [Paka repository](https://github.com/FastVM/paka/tree/main/bench) if you'd like to run them on your machine.
```
Comment (lqd) [inline README.md:120]: Tiny typo here

```suggestion
All benchmarks were run in hyperfine on a `2020 MacBook Air M1` with `8GB RAM` running `Big Sur 11.2.3`. The implementations we benchmark are idiomatic and consistent between target benchmark languages. All benchmarks may be found in the [Paka repository](https://github.com/FastVM/paka/tree/main/bench) if you'd like to run them on your machine.
```

File: README.md (hunk 1 above)
Comment (slightknack) [reply README.md:125]: The ands are intentional, but nice catch on the 'that's!

**Pull request merged**